src/main/java/mif/vu/ikeea/Helper/PaginationHelper.java

```diff
@@ -0,0 +1,35 @@
+package mif.vu.ikeea.Helper;
+
+import mif.vu.ikeea.Payload.FilterGoalRequest;
+import mif.vu.ikeea.Payload.FilterLearningDayRequest;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PaginationHelper {
+
+    Pageable pageable = PageRequest.of(0, 10);
+
+    public Pageable getPageableLearningDay(FilterLearningDayRequest filterLearningDayRequest) {
+
+        if(filterLearningDayRequest.getPage() != null && filterLearningDayRequest.getSize() != null) {
```

Comment:
turėtų būti paprasčiausiai taip:

`
if(filterLearningDayRequest.getPage() == null) {
      page = 0;
}

if (filterLearningDayRequest.getSize() == null) {
  size = 10;
}

return PageRequest.of(page, size);
`